docker/integration.sh

```diff
@@ -8,6 +8,6 @@ fi
 rake package:checkout &&
 rake package:build_gem &&
 rake package:generate_manpages &&
-cd package && osc build $PRODUCT x86_64 --no-verify --trust-all-projects && cd .. &&
+cd package && osc build $PRODUCT x86_64 --no-verify --trust-all-projects --clean && cd .. &&
```

Comment:
Why is this clean needed? Maybe I missed the error in the CI logs.

Reply:
The error was not shown in the CI logs, because the process went interactive and was waiting for a user input.

Reply:
@felixsch like this abborted run: https://ci.suse.de/user/scc/my-views/view/scc-jobs/job/scc-Connect/1321/console 
it just appeared hanging for days.

Reply:
Ah I see.